test/remote/gateways/remote_usa_epay_transaction_test.rb

```diff
@@ -22,6 +22,20 @@ def test_successful_purchase_with_track_data
     assert_success response
   end
 
+  def test_successful_authorization_with_manual_entry
+    @credit_card.manual_entry = true
```

Comment:
Hmmm they pass but I guess that is because the manual_entry parameter is optional. Since credit card is two words should I refactor the test to use `@credit_card`?